src/main/java/com/mkopec/ticketbookingapp/domain/Reservation.java

```diff
@@ -0,0 +1,39 @@
+package com.mkopec.ticketbookingapp.domain;
+
+import lombok.Data;
+
+import javax.persistence.*;
```

Comment:
importy z gwiazdkami nie są preferowane - dobre IDE i tak ci je schowa, a jak chcesz zobaczyć jaka klasa jest importowana to tak nie zobaczysz :(

Reply:
Nie wiedziałem o tym, ale to kwestia ustawień IDE. Teraz już będę pamiętał